web/src/components/CityNotCooperatingFooter.tsx

```diff
@@ -54,8 +45,10 @@ const CityNotCooperatingFooter = ({ languageCode }: CityNotCooperatingFooterProp
     <FooterContainer>
       <StyledIcon src={CityNotCooperatingIcon} />
       <Question>{t('cityNotFound')}</Question>
-      <Button to={pathnameFromRouteInformation({ route: CITY_NOT_COOPERATING_ROUTE, ...{ languageCode } })}>
-        {t('suggestToRegion', { appName: buildConfig().appName })}
+      <Button variant='outlined'>
+        <Link to={pathnameFromRouteInformation({ route: CITY_NOT_COOPERATING_ROUTE, ...{ languageCode } })}>
+          {t('suggestToRegion', { appName: buildConfig().appName })}
+        </Link>
       </Button>
```

Comment:
🤔 Not sure if this is the right approach to show a link in button optics. Let me know if you have objections or better ideas.

Reply:
Not really sadly, with `component="button"` we need to supply an `onClick` function instead of an `href`.